game_scripts/analysis.py

```diff
@@ -292,9 +291,9 @@ def setupGraph(self):
 
         #create graph
         self.cleartempgraphs()
-        print("debug a: "+str(self.feedback_view.final_df))
-        print("debug a: "+str(self.final_df))
-        self.working_graph = ReviewGraph(self.feedback_view.final_df)
+        print("debug a: "+str(self.feedback_view.mol_view.assay_df))
+        print("debug a: "+str(self.feedback_view.mol_view.assay_df))
+        self.working_graph = ReviewGraph(self.feedback_view.mol_view.assay_df)
```

Comment:
change form self.final_df to self.feedback_view.mol_view.assay_df for consitancy and dataflow